common/stacks/src/main/scala/weco/api/stacks/models/SierraItemIdentifier.scala

```diff
@@ -28,7 +29,7 @@ object SierraItemIdentifier {
       ontologyType = "Item"
     )
 
-  def fromSourceIdentifier(
+  def fromOldSourceIdentifier(
```

Comment:
what does `old` mean?

Reply:
"old" is just "the old way of doing things", a change I made to keep the requests API compiling – I didn't want to change too much in one go. The next PR will remove this method when I flip the requests API to use the new types.